docker/Dockerfile

```diff
@@ -123,10 +141,13 @@ RUN set -e; \
         for attempt in $(seq "$DEB_MAX_ATTEMPT" -1 0); do \
             [ "$attempt" -gt 0 ]; \
             sudo DEBIAN_FRONTEND=noninteractive apt-get install -y \
+                $( [ "$CRIS_IMG_ARCH" != 'amd64' ] || echo ' \
```

Comment:
Do not change the sorting order or it'll get hard to manage when the list grows.